proto/axelar/nexus/v1beta1/types.proto

```diff
@@ -42,3 +43,22 @@ message LinkedAddresses {
   axelar.nexus.exported.v1beta1.CrossChainAddress recipient_address = 2
       [ (gogoproto.nullable) = false ];
 }
+
+message RateLimit {
+  string chain = 1
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  cosmos.base.v1beta1.Coin limit = 2 [ (gogoproto.nullable) = false ];
+  google.protobuf.Duration window = 3
+      [ (gogoproto.stdduration) = true, (gogoproto.nullable) = false ];
+}
+
+message TransferRate {
+  string chain = 1
+      [ (gogoproto.casttype) =
+            "github.com/axelarnetwork/axelar-core/x/nexus/exported.ChainName" ];
+  cosmos.base.v1beta1.Coin amount = 2 [ (gogoproto.nullable) = false ];
+  uint64 epoch = 3;
+  bool outgoing = 4; // indicates whether the tracking is for transfers outgoing
```

Comment:
1. If I understand the code correctly, this is not a rate but the running sum during the epoch
2. if the window length changes you could end up reusing epochs

Reply:
I suggest `TransferEpoch`

Reply:
It is tracking the rate of transfers by amount in that window. I don't mind changing the name here, although the msg server `SetTransferRateLimit` still feels more intuitive.